services/minespace-web/src/styles/components/MajorMineApplication.scss

```diff
@@ -12,7 +12,7 @@
   border-top: none;
 }
 
-.page-scroll-wrapper .ant-collapse-content>.ant-collapse-content-box {
+.collapse-no-padding .ant-collapse-content>.ant-collapse-content-box {
```

Comment:
Was testing this page to make sure that I wasn't breaking the rest of the app style and this was sticking out like a sore thumb. The padding around the scrollbar was _not pretty._ Fixed the selector.